articles/how-to-convert-text-to-emoji-using-python/index.md

```diff
@@ -0,0 +1,148 @@
+Try more tricks using the  `dictionary.get()` method.
+Kindly try this yourself during practice to perfect your skills
+To determine how many items a dictionary has, use the `print(len(details))`
+
+### Text to emoji converter
+You may believe that printing emojis would require a complex algorithm. However, let me assure you that this is not the case; in fact, writing code is very easy. This is another cool application of using a dictionary.
+To build this program we start by calling the input function stored in a string. And then we will call the `.split(“ ”)` method. this method goes through this string and anywhere it finds a space it uses it to separate this string into multiple words and return it as a list. Then we will define a dictionary for mapping special characters into emojis features.  We will then add key values of the pair and map it to a specific emoji character.
+You can press ctrl, cmd, and space to bring out the emoji box on your mac system, on your system for windows, press and hold the window button and either the (.) or semicolon(;) until you see the emoji picker appear.
+You will then loop through the string using the `.get()` method.
+See the example below.
+
+```Python
+message = input("> ")
+words = message.split( " ")
+emojis = {
+   ":)" : "😀",
+   ":(" : "😞",
+   "lol" : "😂",
+   "sick":"😨",
+   "happy": "😀",
+   "mermaid": "🧜‍"
+}
+outcome = " "
+for word in words:
+   outcome += emojis.get(word, word) + " "
+print(outcome)
+```
+
+We'll use the define function to transform this code into a reusable function that we can use in a variety of applications.
+
+```Python
+def  emoji_converter(message):
+ 	words = message.split( " ")
+emojis = {
+   ":)" : "😀",
+   ":(" : "😞",
+   "lol" : "😂",
+   "sick":"😨",
+   "happy": "😀",
+   "mermaid": "🧜‍"
+}
+outcome = " "
+for word in words:
+   outcome += emojis.get(word, word) + " "
+return output
+
+
+Message = input  (“>”)
+print(emoji_converter(message) )
+```
+
+Pretty easy, right?
+
+*Try this yourself*
+
+### Conclusion
+Writing codes to convert text to emojis is quite easy to do and requires sound knowledge on the basis of python language. I hope you understand this concept better now and how they work. you can now practice the codes.
+
+### Reference
+ - [pdfdrive](https://www.pdfdrive.com/python-programming-python-programming-for-beginners-python-programming-for-intermediates-e180663309.html)
+- [learn python](https://www.udemy.com/course/learn-python/)
+- [learn python](https://www.learnpython.org/)
+
+### About the author
+![author image](/engineering-education/how-to-convert-text-to-emoji-using-python/majiyebo.jpg)
+Majiyebo Ezra Shewuri
+Majiyebo Ezra is pursuing a degree in medicine and surgery as an undergraduate. Technical writing, copywriting, and python programming are his passions (data science). He started his career as a data scientist two years ago. He is enthralled by new technology. Ezra is also a gamer and a sportsman.
```

Comment:
@majiezra Thank you for submitting your first article. 

For the author details, please create a new folder at the `authors` folder with the following files:
1. `index.md` -- this file carries your bio
2. `avatar.jpg` -- this is your avatar

Feel free to look at other students' folders to get the overview. You can also duplicate one of the author folders and edit it accordingly.

Reply:
thanks for the observation, i will do that now

Reply:
> thanks for the observation, i will do that now

I just tried to do it now, but it would require me to open a new pull request. I think you can do it since you have the admin privileges too.

Reply:
You won't have to open another pull request. Just create the folder and files and commit to the same branch that has this pull request.

Reply:
> You won't have to open another pull request. Just create the folder and files and commit to the same branch that has this pull request.

Alright thanks, done